build-system/tasks/helpers.js

```diff
@@ -75,6 +75,7 @@ const UNMINIFIED_TARGETS = [
   'amp-inabox.js',
   'amp-shadow.js',
   'amp.js',
+  'integration.js',
```

Comment:
This is a partial revert of #24471. I'm not sure why we removed it initially, but per offline discussion with @lannka, this seems ok to reinstate. Filed https://github.com/ampproject/amphtml/issues/25108 to track any follow up refactoring.

Reply:
@cathyxz / @lannka: Are we sure we don't want to add this for minified builds as well for consistency between `gulp build` and gulp dist`? (The minified equivalent is `f.js`.)

I ask because several of our tests are able to switch between `--compiled` mode and regular mode, and I'm concerned that this might break some assumptions in code (not sure what they might be).

Reply:
Got it. I'll add in `f.js` for consistency.

Reply:
This list is now different from the one used by our internal release scripts. (See cl/269449844.)

@lannka @erwinmombay @jridgewell @danielrozenberg Do we want to change that list too? (I don't fully understand how `AMP_CONFIG` is used by the 3p frame.)